scripts/dkms.mkconf

```diff
@@ -82,7 +82,10 @@ DEST_MODULE_LOCATION[3]="/extra/zcommon/zcommon"
 BUILT_MODULE_NAME[4]="zfs"
 BUILT_MODULE_LOCATION[4]="module/zfs/"
 DEST_MODULE_LOCATION[4]="/extra/zfs/zfs"
-BUILT_MODULE_NAME[5]="icp"
-BUILT_MODULE_LOCATION[5]="module/icp/"
-DEST_MODULE_LOCATION[5]="/extra/icp/icp"
+BUILT_MODULE_NAME[6]="icp"
+BUILT_MODULE_LOCATION[6]="module/icp/"
+DEST_MODULE_LOCATION[6]="/extra/icp/icp"
+BUILT_MODULE_NAME[7]="zlua"
+BUILT_MODULE_LOCATION[7]="module/lua/"
+DEST_MODULE_LOCATION[7]="/extra/lua/zlua"
```

Comment:
The updated dkms builders caught this failure.  You need only add new entries for `zlua` and can leave the `icp` ones alone.

```
Loading new zfs-0.7.0 DKMS files...
dkms.conf: Error! No 'BUILT_MODULE_NAME' directive specified for record #5.
dkms.conf: Error! No 'DEST_MODULE_LOCATION' directive specified for record #5.
dkms.conf: Error! Directive 'DEST_MODULE_LOCATION' does not begin with
'/kernel', '/updates', or '/extra' in record #5.
Error! Bad conf file.
```